Tools/autotest/default_params/copter-hexa.parm

```diff
@@ -0,0 +1,94 @@
+EK2_ENABLE      1
```

Comment:
you added a parameter file without referencing it in sim_vehicle.py. Why is it needed at all? The standard copter.param should be fine for hexa.